### ISSUE TITLE
Multiple definitions for businessHours

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2686]
Originally reported on Google Code with ID 2421
```
bi ve seen this been asked over stackoverflow etc quite often. And was really great
to see it being implemented.
However would it be possible to make it one step further?
Instead of applying
this structure
{
    start: '10:00', // a start time (10am in this example)
    end: '18:00', // an end time (6pm in this example)

    dow: [ 1, 2, 3, 4 ]
    // days of week. an array of zero-based day of week integers (0=Sunday)
    // (Monday-Thursday in this example)
}

to be able to apply specific ranges per day? for instance 10-18 1,2,3
8-14 and 17-21:00 4,5,6 etc?
this way it would be possible to make it more customizable
```

Reported by `nikolaspapanik` on 2015-01-14 11:16:17

Imported with **8** stars.

**Comment by MetalWingsWeb** [comment]
How is the status of this issue? I would like to contribute.

**Comment by san983** [comment]
me too :+1:

**Comment by dtmonterrey** [comment]
Well, @arshaw said that he would tackle this pull request https://github.com/fullcalendar/fullcalendar/pull/237 when dealing with this bug report.

There's one person on stackoverflow asking me to update my original fork. I'm redirect him to here.

**Comment by MetalWingsWeb** [comment]
I'll be waiting then for a new status. By the moment, I'm going to use background events to handle with this issue. Thanks @dtmonterrey

**Comment by smartkid77** [comment]
I have been using this feature in my test site. Is that business hours feature won't work on timelineDay view? and also I am getting error of "Uncaught TypeError: spec.class is not a function".

Please someone help me to fix it.

**Comment by arshaw** [comment]
@smartkid77 please use the scheduler issue tracker for that problem. Please make a JSBin recreation before posting. Probably a simple problem of not having the right dependencies.

NOTES TO SELF:
- look at [PR 237](https://github.com/fullcalendar/fullcalendar/pull/237) when tackling this problem
- `eventConstraint` and `selectConstraint` should also accept this array format

**Comment by billmn** [comment]
Any news about this?

**Comment by guiporto** [comment]
hey guys, any news?

**Comment by espen** [comment]
@guiporto @billmn  if there is any news it will be posted here. Use the "Subscribe" button for receiving notifications. Also: this is an open source project so pull requests are welcome.

**Comment by MetalWingsWeb** [comment]
@espen I think that pull requests were sent, but they weren't merged. Let's wait for @arshaw reply about this issue.

**Comment by espen** [comment]
I don't see such a PR @MetalWingsWeb ?

**Comment by MetalWingsWeb** [comment]
@espen they said on the previous comments, if they didn't sent a PR, it's my mistake.

**Comment by arshaw** [comment]
this has been implemented and release in [v2.9.1](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.9.1).

update docs:
http://fullcalendar.io/docs/display/businessHours/
